known issue for when an integration level output is set to default (#1615)
elastic/ingest-docs: docs/en/ingest-management/release-notes/release-notes-8.16.asciidoc

----
====

[discrete]
[[known-issue-206131]]
.Integration output fails when using default output
[%collapsible]
====
*Details* +
Beginning in version 8.16.0 you can specify an output per integration policy. However, setting the integration output to the default creates an invalid output name.

*Impact* +

> Until this issue is resolved, as a workaround you can create a clone of the default output and then set it as the output for an integration policy. Refer to issue link:https://github.com/elastic/kibana/issues/206131[#206131] for details.

Reviewer: ```suggestion
As a workaround, you can create a clone of the default output and then set it as the output for an integration policy. Refer to issue link:https://github.com/elastic/kibana/issues/206131[#206131] for details and status.
```
